client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -288,6 +297,34 @@ function CheckoutSummaryPlanFeatures( props: {
 	);
 }
 
+function CheckoutSummaryChatIfAvailable( props: { siteId: number | undefined } ) {
+	const currentPlan = useSelector( ( state ) =>
+		props.siteId ? getCurrentPlan( state, props.siteId ) : undefined
+	);
+	const translate = useTranslate();
+
+	const billingPeriod = useSelector( getBillingInterval );
+	const isAnnualPeriod = billingPeriod === IntervalLength.ANNUALLY;
+
+	const currentPlanSlug = currentPlan?.productSlug;
+
+	const isChatAvailable =
+		currentPlanSlug &&
+		( isWpComPremiumPlan( currentPlanSlug ) ||
+			isWpComBusinessPlan( currentPlanSlug ) ||
+			isWpComEcommercePlan( currentPlanSlug ) ) &&
+		isAnnualPeriod;
```

Comment:
Live chat support is depending on the plan billing period not on the `getBillingInterval` which is what user has selected in the Plugins UI

![CleanShot 2022-02-22 at 11 31 00@2x](https://user-images.githubusercontent.com/12430020/155103391-e3c18a90-8c0f-479a-8e0f-8ed89956533a.png)

Reply:
Thanks for your comments @cpapazoglou . 

I thought about that condition but the reason why I have used the billing interval _of the plugin_ as opposed to the _plan_ billing period is that when purchasing a plugin, the `Live chat support` is displayed depending on the _plugin_ billing interval 
|Monthly|Annual|
|--|--|
|![CleanShot 2022-02-22 at 10 43 28@2x](https://user-images.githubusercontent.com/3519124/155106230-95fbcfbe-7299-450b-83ca-f39772184a4e.png)|![CleanShot 2022-02-22 at 10 44 49@2x](https://user-images.githubusercontent.com/3519124/155106089-f7bd0da1-548d-45bf-a46d-af1ef5ff2251.png)|

So it seems to be inconsistent with the change above, what do you think about it?

Reply:
That seems like a bug we need also to address, my mistake 🙈. 

We should depend on the plugin billing interval only when user needs to also upgrade their plan and thus we add a plan to their cart. On the rest of the cases, we should depend on the plan interval.

Reply:
No worries, I should have clarified the change in the Plugin details page.

I will update the code and the PR to address your comments and create an issue for the Plugin details page message described in the above screenshots.

Reply:
Bug #61259 created